drivers/hmis/app/models/hmis/hud/processors/base.rb

```diff
@@ -290,6 +290,28 @@ def normalize_single_custom_field_value(cded, value)
       else
         raise "unexpected value \"#{value}\""
       end
+    when 'file'
+      case value
+      when String
+        # If the value is a string, it should be a blob ID for a file that's just been uploaded to ActiveStorage,
+        # but doesn't yet have a File record in our database.
+        blob = ActiveStorage::Blob.find_by(id: value)
+        file = Hmis::File.new
+        file.name = blob.filename
+        file.client = @processor.send(:client_factory)
+        file.enrollment = @processor.send(:enrollment_factory)
+        file.client_file.attach(blob)
+        file.user = @processor.current_user
+        file.updated_by = @processor.current_user
+        file
+      when Object
+        # If the value is an Object, it should be a File record in our db
+        raise "unexpected value \"#{value}\"" unless value.key?('fileBlobId') && value.key?('id')
+
+        Hmis::File.find(value['id'])
```

Comment:
Should the File's `updated_by_user_id` change to point at a user who submits the assessment but doesn’t change the file? Rn the behavior is to update the CDE updated date without changing the file itself doesn’t change.

Reply:
No, I don't think the updated_by_user_id on the file should be updated when the assessment is re-submitted